OpenRA.Mods.Common/Widgets/Logic/Settings/HotkeysSettingsLogic.cs

```diff
@@ -215,7 +219,7 @@ void InitHotkeyRemapDialog(Widget panel)
 
 			var duplicateNotice = panel.Get<LabelWidget>("DUPLICATE_NOTICE");
 			duplicateNotice.TextColor = ChromeMetrics.Get<Color>("NoticeErrorColor");
-			duplicateNotice.IsVisible = () => !isHotkeyValid;
+			duplicateNotice.IsVisible = () => !isHotkeyValid && !selectedHotkeyDefinition.Readonly;
```

Comment:
`isHotkeyValid` gets set to `duplicateHotkeyDefinition == null || selectedHotkeyDefinition.Readonly`, so is the new condition here really necessary? (Same for all the additions below.)

Reply:
Nice catch! This was leftover from an earlier iteration. Fixed.